components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -14,6 +14,30 @@
 
 #import <UIKit/UIKit.h>
 
+@class MDCTabBarView;
+
+/**
+ Delegate protocol for MDCTabBarView. Clients may implement this protocol to receive notifications
+ of selection changes by user action in the tab bar.
+ */
+@protocol MDCTabBarViewDelegate <UIScrollViewDelegate>
+
+@optional
+
+/**
+ Called before the selected item changes by user action. Return YES to allow the selection. If not
```

Comment:
According to UIKit, the comment can be updated as "Asks the delegate whether the specified item should be selected."

Reply:
Done.